gef-ui/server/services/api.js

```diff
@@ -177,18 +177,45 @@ const deleteFacility = async ({ facilityId, userToken }) => {
   }
 };
 
-const getCompaniesHouseDetails = async ({ companyRegNumber, userToken }) => {
-  if (!isValidCompaniesHouseNumber(companyRegNumber)) {
-    console.error('getCompaniesHouseDetails: API call failed for companyRegNumber %s', companyRegNumber);
-    throw new Error('Invalid company house number');
-  }
-
+const getCompanyByRegistrationNumber = async ({ registrationNumber, userToken }) => {
   try {
-    const { data } = await Axios.get(`/gef/company/${companyRegNumber}`, config(userToken));
-    return data;
+    if (!registrationNumber) {
+      return {
+        errRef: 'regNumber',
+        errMsg: 'Enter a Companies House registration number.',
+      };
+    }
+
+    if (registrationNumber && !isValidCompaniesHouseNumber(registrationNumber)) {
```

Comment:
Probably don't have to check for `registrationNumber` here as we've done it on linke 182, so we know its truthy? :)

Reply:
Good spot, changed :)